docs/pages/blog/meet-joy.md

```diff
@@ -0,0 +1,60 @@
+
+## Why not just build a different theme?
+
+One of the things we're optimizing for with this new product is to break away from Material Design. A different theme would still require you and your team to have at least some sense of how Material Design is structured and how that is expressed into Material UI in order to reach a fresher look. We wanted to provide an alternative to that. With Joy, we want you to be able to start, by default, with a fresher design.
+
+Additionally, we had many ideas for improving the experience of developing design systems and a separate package is an opportunity to prototype and test some of these new features without running the risk of potentially making any mistakes and causing inconveniences to hundreds of teams that use Material UI today.
+
+Material UI and Joy UI will definitely share most of the conventions, features, and overall philosophy. This will not only decrease the learning curve for any of you who are willing to try but also because we think of the two containing our take on the ideal developer experience for developing design systems. So, if at any time, Joy UI has something that Material UI doesn't, the catch-up will at some point happen, and the same goes for the other way around.
+
+## What are the new things we're introducing in Joy UI?
+
+Aside from a fresher look and feel for your upcoming project, here are a few features we're planning for the alpha version of Joy UI:
+
+### Global variants
+
+Instead of defining variants at the component level, Joy defines variants at the global level. This sounds bold at first, but the more we experiment the more it starts to make sense. Global variants mostly affect three CSS properties: color, background, and border. What this means is that you'll be able to toggle between the same set of variants in any component, allowing for easier consistency across your app. The four variants are text, outlined, soft, and solid.
+
+### Component integration
+
+We put a lot of effort into designing CSS variables for each component so that they can be seamlessly composed with the least customization.
+One good example is the card component, where the border radius of the card’s children automatically adapts to the card.
+These small details make the component resilient to different scenarios which saves tons of time for manual adjustments.
+Joy does not sacrifice the customization experience in order to have the automatic adjustment.
+You are able to override the style completely via the usual CSS or even adjust the predefined CSS variables from the `sx` prop.
+
+### Perfect dark mode for server-side rendering
+
+Joy provides a simple and effective way to prevent UI flickering when users refresh or re-enter the page after they have toggled dark mode on.
+Moreover, it takes care of the synchronization between browser tabs with a single line of code which allows developers to configure attributes and settings as required.
```

Comment:
```suggestion
What's more, it takes care of the synchronization between browser tabs with a single line of code, which allows developers to configure attributes and settings as required.
```

> which allows developers to configure attributes and settings as required.

I don't understand this. What's the current limitation?

Reply:
@siriwatknp could you help clarify?!